ui/apps/platform/src/Containers/Risk/RiskSidePanel.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { Alert, Bullseye, Spinner } from '@patternfly/react-core';
+
+import { fetchDeploymentWithRisk } from 'services/DeploymentsService';
+import type { DeploymentWithRisk } from 'services/DeploymentsService';
+
+import CloseButton from 'Components/CloseButton';
+import { PanelNew, PanelBody, PanelHead, PanelHeadEnd, PanelTitle } from 'Components/Panel';
+
+import RiskSidePanelTabs from './RiskSidePanelTabs';
+
+export type RiskSidePanelProps = {
+    selectedDeploymentId: string;
+    setSelectedDeploymentId: (string) => void;
+};
+
+function RiskSidePanel({ selectedDeploymentId, setSelectedDeploymentId }: RiskSidePanelProps) {
+    const [selectedDeployment, setSelectedDeployment] = useState<DeploymentWithRisk | null>(null);
+    const [isFetching, setIsFetching] = useState(false);
+
+    useEffect(() => {
+        setIsFetching(true);
+        fetchDeploymentWithRisk(selectedDeploymentId)
+            .then((deploymentWithRisk) => {
+                setSelectedDeployment(deploymentWithRisk);
+            })
+            .catch(() => {
+                setSelectedDeployment(null);
```

Comment:
I'm a little surprised typescript isn't complaining here considering how it's defined in `RiskSidePanelProps`